gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/test/java/io/gravitee/rest/api/management/v2/rest/mapper/ApiMapperTest.java

```diff
@@ -69,7 +69,7 @@ void shouldMapToUpdateApiEntityV2() {
         assertThat(updateApiEntity.getCrossId()).isNull();
         assertThat(updateApiEntity.getName()).isEqualTo(updateApi.getName());
         assertThat(updateApiEntity.getVersion()).isEqualTo(updateApi.getApiVersion());
-        assertThat(updateApiEntity.getGraviteeDefinitionVersion()).isEqualTo(updateApi.getDefinitionVersion().name());
+        assertThat(updateApiEntity.getGraviteeDefinitionVersion()).isEqualTo(ApiMapper.INSTANCE.mapFromDefinitionVersion(updateApi.getDefinitionVersion()));
```

Comment:
Suggestion:
As we are in a specific "V2" use case, and this value is not going to change in the future, I would propose to directly compare to the enum label.
```suggestion
        assertThat(updateApiEntity.getGraviteeDefinitionVersion()).isEqualTo(ApiMapper.INSTANCE.mapFromDefinitionVersion(DefinitionVersion.V2.getLabel()));
```

Thought:
On top of that, the test you've written will always be `true` since `ApiMapper.INSTANCE.mapFromDefinitionVersion(...)` is the function that has been called in `apiMapper.map(...)`